Rakefile

```diff
@@ -217,6 +217,33 @@ namespace :release do
        "Groonga #{version}!!!")
     sh("git", "push", "origin", "v#{version}")
   end
+
+  desc "Post release announces"
+  task :announce do
+    latest_news = Dir.glob("doc/source/news/*.*").max do |a, b|
+      File.basename(a).to_f - File.basename(b).to_f
+    end
+    latest_release_note = File.read(latest_news).split(/^## /)[1]
+    latest_release_note_version = latest_release_note.lines.first[/[\d.]+/]
+    if latest_release_note_version != version
+      raise "release note isn't written"
+    end
+    latest_release_summary =
+      latest_release_note[/^### Summary\s*(.+?)(?=^###|\z)/m, 1]
```

Comment:
Can we use the "This is our annual major release! This release doesn't have any backward incompatible changes! So you can upgrade Groonga without migrating your existing databases. You can still use your existing databases as-is." part in https://github.com/groonga/groonga/blob/main/doc/source/news/15.md#release-1500---2025-02-09 instead of creating "Summary" section?

Reply:
fix: [11cd247](https://github.com/groonga/groonga/pull/2291/commits/11cd24740d9a5432a57a4552248b30cbe371985f) Sure. It sounds better for me too!